client/src/components/Site.vue

```diff
@@ -86,13 +102,13 @@
     </section>
 
     <section class="section section-xsmall ">
-      <h4 class="is-size-4">Sky - Ambient Temp
-        <a class="helptoggle is-pulled-right"><sup><small>?</small></sup></a><span class="help is-pulled-right">Sky Temperature is inferred from 8-14µm irradiance measure by a Boltwood II cloud sensor at the site's weather station.</span>
+      <h4 class="is-size-4">Sky Transparency*
```

Comment:
What is the asterisk for?

Reply:
Basically just to show that the sky transparency is a calculated value and it isn't measured directly. (see task 4: *Switch sky-ambient plot to be transparency but make it clear it is a calculated value not measured directly, or plot both sky-ambient and transparency on same graph. Add threshold line in red.*) 

I wasn't quite sure what would be the best way to distinguish the sky transparency thing as being a calculated value. Should I just mention that fact in the dropdown instead? Or color the dropdown icon differently or something? The asterisk was just the first thing that came to mind.